statsforecast/models.py

```diff
@@ -4937,7 +4937,7 @@ def predict(
         res = self.trend_forecaster.predict(**kwargs)
         seas = _predict_mstl_seas(self.model_, h=h, season_length=self.season_length)
         res = {key: val + seas for key, val in res.items()}
-        if level is None:
+        if level is None or f'lo-{level[0]}' in res:
```

Comment:
Maybe we could add more robust handling, I think that for example if `self.trend_forecaster.prediction_intervals is not None` and a user passes `level` then the model will not return the conformal intervals but the prediction intervals of the underlying model, because `trend_forecaster` will always return prediction intervals if `level is not None` and it can handle `level`. 

Adding:

```python
add_underlying_level = "level" in signature(self.trend_forecaster.predict).parameters and (self.trend_forecaster.prediction_intervals is not None)
if add_underlying_level:
     kwargs["level"] = level
```

could solve the problem. This way `f'lo-{level[0]}' in res` will be false if the user requests conformal intervals (`self.trend_forecaster.prediction_intervals is not None`).